background.html

```diff
@@ -285,7 +285,10 @@ <h4>{{ requestReceivedTitle }}</h4>
         <h4>{{ requestAcceptedTitle }}</h4>
         <p class="secretWords"></p>
         <p class="transmissionStatus">Please be patient...</p>
-        <div class='buttons'>
+        <div id="deviceAliasView" style="display: none;" >
+          <input id="deviceAlias" type="text" placeholder="Device Alias" required>
+        </div>
+          <div class='buttons'>
```

Comment:
```suggestion
        <div class='buttons'>
```

Reply:
This is basically the jungle, outside the linter's jurisdiction...
Good catch :)